mofacts/client/views/experiment/card.js

```diff
@@ -1768,7 +1766,7 @@ async function unitIsFinished(reason) {
 function getButtonTrial() {
   const curUnit = Session.get('currentTdfUnit');
   // Default to value given in the unit
-  let isButtonTrial = 'true' === _.chain(curUnit).prop('buttontrial').first().trim().value().toLowerCase();
+  let isButtonTrial = true === eval(curUnit.buttontrial);
```

Comment:
Don't do evals, ever.  It's a massive security implication.  Better to do a string comparison boolean operation. Also users may not type it correctly cased to eval correctly